DataStructures/Graphs/FloydWarshall.java

```diff
@@ -14,7 +14,11 @@ public FloydWarshall(int numberofvertices) {
             [numberofvertices
                 + 1]; // stores the value of distance from all the possible path form the source
     // vertex to destination vertex
-    Arrays.fill(DistanceMatrix, 0);
+    for(int i = 0;i < numberofvertices + 1; i++) {
```

Comment:
I don't think you need to fill `DistanceMatrix` with 0s as it will already be initialized with it when creating with `new` keyword. 

So, for example

```java
int[][] DistanceMatrix = new int[2][2];

// Elements of DistanceMatrix will be [[0,0],[0,0]] by default
```

Please check it and do needful.

Reply:
Well, I just patched up the problem with Arrays.fill() [Maybe the author just wanted to emphasize that the array is filled with 0s? Especially because this an algorithm repo and beginner coders also use this ]
I encourage you to open up this issue, and see if this works! :)

Reply:
Ok @icoder211 , that seems valid but I guess, instead of filling array just for the sake of it, it would be good to add a comment above its declaration stating the fact that it will be initialized by 0s as default, so that newbies or anyone who didn't know it, will obtain some useful knowledge about it. 

This is just what I think should be like for making others learn with help of your contributions. Its perfectly fine to disagree. 

I hope you take it as positive or constructive criticism.

Reply:
Agreed, for loops aren't necessary. Thanks!